.github/workflows/ci.yml

```diff
@@ -30,10 +30,11 @@ permissions:
 defaults:
   run:
     shell: bash
+concurrency:
+  group: "${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}"
```

Comment:
Does this cancel parallel try builds? Right now we allow infinite concurrent try builds, and IMO we should continue to do so.

Reply:
Already discussed [here](https://github.com/rust-lang/rust/pull/112955#discussion_r1240967898). I hope it shouldn't cancel try builds, because they should have different SHAs.

Reply:
`github.ref` is not the commit SHA, it's `refs/heads/trying`.

Reply:
Then that's bad, I will investigate and fix. Thank you!

Reply:
Yeah it cancels parallel try builds, dammit. Sorry for that, I will send a PR with a fix soon.